[OSD][airgap] Fails with `cannot install` because both channel and revision are specified

Currently running an offline deployment and the deployment fails due to:
```
2024-12-12 16:30:19 DEBUG unit.opensearch-dashboards/0.juju-log server.go:325 restart:2: Re-emitting deferred event <InstallEvent via OpensearchDasboardsCharm/on/install[1]>.                                                                                        
2024-12-12 16:30:19 INFO unit.opensearch-dashboards/0.juju-log server.go:325 restart:2: Installing snap opensearch-dashboards, revision 24, tracking edge                                                                                                             
2024-12-12 16:30:19 WARNING unit.opensearch-dashboards/0.restart-relation-created logger.go:60 error: cannot install "opensearch-dashboards": Both revision and channel are                                                                                           
2024-12-12 16:30:19 WARNING unit.opensearch-dashboards/0.restart-relation-created logger.go:60        present in the request for this Snap. It should be one or the other                                                                                             
2024-12-12 16:30:19 WARNING unit.opensearch-dashboards/0.restart-relation-created logger.go:60        exclusively.                 
2024-12-12 16:30:19 ERROR unit.opensearch-dashboards/0.juju-log server.go:325 restart:2: Snap: 'opensearch-dashboards'; command ['snap', 'install', 'opensearch-dashboards', '--channel="edge"', '--revision="24"'] failed with output = '' 
```

Indeed, both are currently specified in our charm.

Current snap, including snapd version, are deployed:
```
root@juju-405c2c-isolated-kafka-airgapped-model-1:/var/lib/juju/agents/unit-opensearch-dashboards-0/charm# snap list 
Name    Version        Rev    Tracking       Publisher   Notes                                                       
core20  20240416       2318   latest/stable  canonical✓  base                                                        
lxd     5.0.3-80aeff7  29351  5.0/stable/…   canonical✓  -                                                           
snapd   2.63           21759  latest/stable  canonical✓  snapd                                                       
```

Looking into other charms, neither [kafka](https://github.com/canonical/kafka-operator/blob/1c6b435abdbba30f6a0ca328ce8f0f555d248767/src/workload.py#L130) nor [opensearch](https://github.com/canonical/opensearch-operator/blob/44162927148aeb3032f9d6334f3a42731eb8d188/src/opensearch.py#L66) specify both at install.

--- a/src/workload.py
+++ b/src/workload.py
@@ -134,7 +134,6 @@ class ODWorkload(WorkloadBase):
             dashboards.ensure(
                 snap.SnapState.Present,
                 revision=OPENSEARCH_DASHBOARDS_SNAP_REVISION,
-                channel="edge",
             )
 
             self.dashboards = dashboards

